app/policies/bar_policy.rb

```diff
@@ -10,11 +10,11 @@ def show?
   end
 
   def create?
-    true
+    record.user == user
   end
 
   def update?
-    record.owner == user
+    true
```

Comment:
Hmm, t'es sûr de vouloir changer ca ?
Anybody should create
Only owners should update